docs/pages/tooltip.js

```diff
@@ -221,9 +211,9 @@ When using Tooltip with [IconButton](/iconbutton), avoid repetitive labeling. Th
           <MainSection.Card
```

Comment:
@ayeshakmaz Should we review the guidance on line 193? 

`When using Tooltip with [IconButton](/iconbutton), avoid repetitive labeling. The \`accessibilityLabel\` provided to IconButton should describe the intent of the button, not the icon itself. For instance, use “Settings” instead of “Cog icon”. Tooltip \`text\` can expand upon that intention, as seen with the \`cog\` IconButton. If Tooltip \`text\` and IconButton \`accessibilityLabel\` contain the same content, pass an empty string to \`accessibilityLabel\`, as seen with the \`send\` IconButton.`}`

Reply:
@ayeshakmaz Is this redundancy an accepted practice now?
<img width="460" alt="Screen Shot 2022-01-26 at 10 35 49 PM" src="https://user-images.githubusercontent.com/10593890/151251035-6cba21b7-c342-4ee3-afc8-b7e8f6a39468.png">

Reply:
Ahh yes I need to update that. For now, the redundancy is expected

Reply:
Actually, I'll update the docs with how to avoid the redundancy now